docs/minikube-walkthrough.md

```diff
@@ -0,0 +1,117 @@
+# Minikube walkthrough 
+
+This document will take you through setting up and trying the sample apiserver on a local minikube from a fresh clone of this repo. 
+
+## Pre requisites
+
+- Go 1.7.x or later installed and setup. More infomration can be found at [go installation](https://golang.org/doc/install)
+- Dockerhub account to push the image to [Dockerhub](https://hub.docker.com/)
+
+## Install Minikube
+
+Minikube is a single node Kubernetes cluster that runs on your local machine. The Minikube docs have installation instructions for your OS. 
+- [minikube installation](https://github.com/kubernetes/minikube#installation)
+
+
+## Clone the repository
+
+In order to build the sample apiserver image we will need to build the apiserver binary. 
+
+```
+cd $GOPATH/src
+mkdir -p github.com/kubernetes
+cd github.com/kubernetes
+git clone https://github.com/kubernetes/sample-apiserver.git
+``` 
+
+## Rename the package imports
+
+After cloning from the repo, the imports will be referencing ```k8s.io/sample-apiserver``` these need to be changed to ```github.com/kubernetes/sample-apiserver```. Recommend you do this with your favourite editor and a find all and replace.
```

Comment:
This shouldn't be necessary if we describe how to check it out into $GOPATH/src/k8s.io/sample-apiserver.

Reply:
good point I will make that change